docs/uml/main/input/CharacteristicDatamodelConcept.puml

```diff
@@ -33,7 +33,7 @@ package models {
                     + {static} REQUIRED_FORMAT: String
                     - x: ComparableQuantity<A>
                     - y: ComparableQuantity<O>
-                    + deSerialize(int): String
+                    + serialize(int): String
```

Comment:
You just removed the Integer parameter with this PR :)

Reply:
I am not quite sure what's to be done here

Reply:
@vickybung1 Please remove the `int` part so that it looks like the method signature in line 46